temporalcli/commands.operator_namespace_test.go

```diff
@@ -0,0 +1,204 @@
+package temporalcli_test
+
+import (
+	"encoding/json"
+	"time"
+
+	"github.com/temporalio/cli/temporalcli"
+	"go.temporal.io/api/enums/v1"
+	"go.temporal.io/api/workflowservice/v1"
+)
+
+func (s *SharedServerSuite) TestOperator_Namespace_Create_List_And_Describe() {
```

Comment:
```suggestion
func (s *SharedServerSuite) TestOperator_Namespace_CreateListAndDescribe() {
```

No need to underscore every word IMO (but pedantic, no need to change if you don't want)